src/modules/DongleDriver.ts

```diff
@@ -192,17 +206,27 @@ export class DongleDriver extends EventEmitter {
       nightMode: _nightMode,
       boxName: _boxName,
       audioTransferMode,
+      wifiType,
+      micType,
     } = config
     const initMessages = [
       new SendNumber(_dpi, FileAddress.DPI),
       new SendOpen(config),
+      new SendBoolean(
+        config.platform === 'android-auto',
+        FileAddress.ANDROID_WORK_MODE,
+      ),
```

Comment:
I think this should be nullable, and we should send nothing unless this is set.

I tested the changes and Android doesn't work unless `platform` is set to `android-auto`. Previously, both Android Auto and Apple Carplay worked without specifying a platform

Reply:
done